gtsam/inference/Conditional.h

```diff
@@ -34,9 +34,10 @@ namespace gtsam {
    * `logProbability` is the main methods that need to be implemented in derived
    * classes. These two methods relate to the `error` method in the factor by:
    *   probability(x) = k exp(-error(x))
-   * where k is a normalization constant making \int probability(x) == 1.0, and
-   *   logProbability(x) = K - error(x)
-   * i.e., K = log(K). The normalization constant K is assumed to *not* depend
+   * where k is a normalization constant making
+   * \int probability(x) = \int k exp(-error(x)) == 1.0, and
+   * logProbability(x) = -(K + error(x))
+   * i.e., K = -log(k). The normalization constant k is assumed to *not* depend
```

Comment:
@dellaert this comment is why I think `negLogConstant` is also suitable. If you think `errorConstant` is better, then I'm good to go.